nixos/doc/manual/release-notes/rl-2103.xml

```diff
@@ -466,6 +466,21 @@ self: super:
      ALSA OSS emulation (<varname>sound.enableOSSEmulation</varname>) is now disabled by default.
     </para>
    </listitem>
+   <listitem>
+    <para>
+     Adobe Flash Player support has been dropped from the tree. In particular,
+     the following packages no longer support it:
+     <itemizedlist>
+      <listitem><package>chromium</package></listitem>
+      <listitem><package>firefox</package></listitem>
+      <listitem><package>qt48</package></listitem>
+      <listitem><package>qt5.qtwebkit</package></listitem>
```

Comment:
[List items](https://tdg.docbook.org/tdg/5.1/listitem.html) need to contain a paragraph of sorts.

```suggestion
      <listitem><simpara><package>chromium</package></simpara></listitem>
      <listitem><simpara><package>firefox</package></simpara></listitem>
      <listitem><simpara><package>qt48</package></simpara></listitem>
      <listitem><simpara><package>qt5.qtwebkit</package></simpara></listitem>
```

Reply:
Fixed!